lib/hanami/cli_base.rb

```diff
@@ -27,5 +27,20 @@ module CliBase
     def define_commands(&blk)
       class_eval(&blk) if block_given?
     end
+
+    def banner(command, nspace = true, subcommand = false)
+      super(command, nspace, namespace != 'hanami:cli')
+    end
+
+    def handle_argument_error(command, error, args, arity)
```

Comment:
@jodosha This is necessary due to a [bug](https://github.com/erikhuda/thor/pull/464) on Thor. We should probably remove after the new version release.

[thor/base.rb#L479](https://github.com/erikhuda/thor/blob/v0.19.4/lib/thor/base.rb#L479) doesn't consider subcommand names to build error messages